docs/usage.rst

```diff
@@ -80,6 +79,29 @@ Every contract will always have the following key/value pairs:
 
 Each contract type defines additional options that you supply here.
 
+TOML configuration
+------------------
+
+The toml configuration is very similar to the others with a few differences:
```

Comment:
Can we refer to TOML consistently in uppercase?

Reply:
done